src/BaseCommand.js

```diff
@@ -10,73 +10,87 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const { Command, flags } = require('@oclif/command')
-const fs = require('fs-extra')
-const chalk = require('chalk')
-const coreConfig = require('@adobe/aio-lib-core-config')
-const DEFAULT_LAUNCH_PREFIX = 'https://experience.adobe.com/?devMode=true#/custom-apps/?localDevUrl='
-const loadConfig = require('./lib/config-loader')
+import { Command, Flags } from '@oclif/core'
+import config from '@adobe/aio-lib-core-config'
+import AioLogger from '@adobe/aio-lib-core-logging'
 
-const { getToken } = require('@adobe/aio-lib-ims')
-const { CLI } = require('@adobe/aio-lib-ims/src/context')
+import { CONFIG_STATE_REGION } from './constants.js'
+import chalk from 'chalk'
 
-class BaseCommand extends Command {
-  getAppConfig () {
-    if (!this.appConfig) {
-      this.appConfig = loadConfig()
-    }
-    return this.appConfig
-  }
+export class BaseCommand extends Command {
+  async init () {
+    await super.init()
 
-  // this was added to get a login context, or force a login if it is not present
-  // this is the only change that was made to this file, otherwise copied from aio-cli-plugin-app/
-  async getAccessTokenAndOrgId () {
-    const creds = {
-      accessToken: await getToken(CLI),
-      ims_org_id: coreConfig.get('project.org.ims_org_id')
-    } // note: looks like we can just use: config.imsOrgId for this
-    return creds
-  }
+    // setup debug logger
+    const command = this.constructor.name.toLowerCase() // hacky but convenient
+    this.debugLogger = AioLogger(
+      `aio:app:state:${command}`,
```

Comment:
Should this be aio:app:storage now?

Reply:
ah I'd say no because the command is still app:state